digiroad2-api-common/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Context.scala

```diff
@@ -246,8 +246,8 @@ class TrafficSignCreateAssets(trafficSignManager: TrafficSignManager) extends Ac
 
 class TrafficSignExpireAssets(trafficSignService: TrafficSignService, trafficSignManager: TrafficSignManager) extends Actor {
   def receive = {
-    case x: Long => trafficSignService.getTrafficType(x) match {
-      case Some(trafficType) => trafficSignManager.deleteAssets(x, trafficType)
+    case x: Long => trafficSignService.getPersistedAssetsByIdsWithExpire(Set(x)).headOption match {
```

Comment:
Automatic process related code, who is the expert on this? Should we include it?

Reply:
I think yes because Manoeuvres continue using this!

Reply:
I think so, but is better check. Manoeuvre is using the actor, and for that is only allow create and delete